### ISSUE TITLE
Xteve keeps deactivating channels...

### ISSUE THREAD

**Issue by flyize** [xteve-project/xTeVe#328]
I have an IPTV provider with a weird situation. For a few of the channels, the channel name gets changed to show whats on. Every time that happens, I think Xteve is deactivating the channel. Could that be the case?

If so, I noticed the tvg-id is blank. If that was set statically, and then the channel name changed, would it still get deactivated?

**Comment by MCTyler** [comment]
Yup, If YOUR PROVIDER changes the channel name, it changes the Unique ID. Ask your Provider to quit doing that.

**Comment by flyize** [comment]
Is your mention of 'unique id' specific to Xteve and different from the TVG-ID?

**Comment by MCTyler** [comment]
> Is your mention of 'unique id' specific to Xteve and different from the TVG-ID?

The channel ID comes from the Provider. For PPV channels, they constantly change the channel ID and it deactivates the channels because that is what the Db links on for mapping.

**Comment by MCTyler** [comment]
We do have a script on Discord that will hash the Stream URL as a CUID, but I have found that Providers also change the URL, so this doesn't work either.

**Comment by flyize** [comment]
Oh that should actually work! Can you please shoot me a link? I don't see it listed in the README

**Comment by MCTyler** [comment]
https://discord.gg/KD2jy5V
